lib/eex/lib/eex/engine.ex

```diff
@@ -18,7 +18,12 @@ defmodule EEx.Engine do
 
       The marker is what follows exactly after `<%`. For example,
       `<% foo %>` has an empty marker, but `<%= foo %>` has `"="`
-      as marker. The allowed markers so far are: `""` and `"="`.
+      as marker. The allowed markers so far are: `""`, `"="`, `"/"`
```

Comment:
Since the markers are four now, can we list them with bullet points?

```
The allowed markers so far are:

  * `""`
  * `"="`
  * `"/"`
  * `"|"`
```

Wdyt?